### PR TITLE
CL handle empty podspec with only image name defined

### PR DESCRIPTION
**Pull request by sjug**
Allows users to specify only pod image location or full podspec.

### CODE REVIEW

File: test/extended/cluster/cl.go [hunk 1]
```diff
@@ -163,7 +167,7 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 					}
 					// TODO sjug: pass label via config
 					labels := map[string]string{"purpose": "test"}
-					err = CreatePods(c, pod.Basename, nsName, labels, config.Spec, pod.Number, tuning, &pod.Sync)
+					err = pod.CreatePods(c, nsName, labels, config.Spec, tuning)
```
Comment (jmencak) [inline test/extended/cluster/cl.go]: Good turning this into a method.

File: test/extended/cluster/cl.go [hunk 2]
```diff
@@ -254,7 +258,7 @@ func mkPath(filename, config string) (string, error) {
 	}
 	// Handle an empty filename.
 	if filename == "" {
-		return "", fmt.Errorf("No template file defined!\n")
+		return "", fmt.Errorf("no template file defined!")
```
Comment (jmencak) [inline test/extended/cluster/cl.go]: Agree with removing '\n', not so sure about '^A-Z' -> '^a-z', but don't care that much. However, if we're changing this at some place, we should be consistent and the same rules should apply across all of cluster-loader.  Perhaps a follow-up cleanup PR if you don't want to do this in this one?

**Comment by jmencak** [comment]
/lgtm

Please remove the /hold if you're planning to address the formatting of the rest of the error messages in a separate PR rather than this one.

File: test/extended/cluster/cl.go (hunk 2 above)
Comment (sjug) [reply test/extended/cluster/cl.go]: Ah the reason was that golint said to update it, I will change the rest as well. Thanks.

**Comment by sjug** [comment]
@jmencak I've edited all the remaining incorrect capitalized `Errorf` messages. PTAL

**Comment by jmencak** [comment]
/hold cancel
/lgtm
There's actually more `e2e.Logf("[A-Z]".*\n"` in the code.  Unblocking for this to merge, but please consider a cleanup PR.

**Pull request merged**